python/dolfinx/fem/function.py

```diff
@@ -428,6 +430,7 @@ def interpolate(
             u: The function, Expression or Function to interpolate.
             cells: The cells to interpolate over. If `None` then all
                 cells are interpolated over.
+            cell_map: Mapping from cells in receiving mesh to cells in `u` mesh
```

Comment:
What's the "receiving mesh"? Presumably `cell_map` maps `cells` to the cells in the `u` mesh?